src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,33 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import './TodoInfo.scss';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      {
+        user && <UserInfo user={user} />
+      }
```

Comment:
```suggestion
      {user && (
        <UserInfo user={user} />
      )}
```
It's wrapped in such a way in real projects.